packages/shared/src/components/RankProgress.tsx

```diff
@@ -162,74 +162,80 @@ export function RankProgress({
     const firstAnimationDuration = 400;
     const maxScale = 1.666;
 
-    const progressAnimation = showRadialProgress
-      ? progressRef.current.animate(
```

Comment:
Probably the optional chaining would work well here too

Reply:
Although would probably work, I think this is safer down the line to exclude any weird things.
Let me know what you think?

Reply:
Well that is quite useful in this cases at least when you're just checking down the chain if any of the parts is null/undefined and can make code clearer in this case. I try to use when I can.
But don't have any 100% strong opinion of using either way if we don't have any standard set.
In this case for example with it would allow you to add an extra check for progressRef as well `progressRef?.current?.animate()`

Reply:
Alright, moving to optional chaining :) 
We don't need the `?.current` thought as it will exist as we use refs.